internal/controller/pkg/revision/deployment.go

```diff
@@ -269,3 +273,15 @@ func buildProviderDeployment(provider *pkgmetav1.Provider, revision v1.PackageRe
 	}
 	return s, d, svc
 }
+
+// filterAnnotations excludes annotations that are meaningless or unsafe to copy
+func filterAnnotations(in map[string]string) map[string]string {
+	ret := make(map[string]string)
+	for label, value := range in {
+		if strings.HasPrefix(label, "kustomize.toolkit.fluxcd.io/") || strings.HasPrefix(label, "kubectl.kubernetes.io/") {
```

Comment:
Does it make sense to make these a "constant" (i.e. a `var doNotPropagate []string = ...`)?

Reply:
Following up here. I think this is mergeable if you have time to address this. :)

Reply:
@negz sorry, busy times! doNotPropagate at function level, or package?

Reply:
I was thinking package (sorry also for the huge delay in response times).

Reply:
@negz At least we have a stable cadence :)